platform-sdk/swirlds-base/src/main/java/com/swirlds/base/time/Time.java

```diff
@@ -45,5 +47,16 @@ public interface Time {
      *
      * @return the curren time relative to the epoch
      */
+    @NonNull
     Instant now();
+
+    /**
+     * Returns a {@link Time} instance
+     *
+     * @return a {@link Time} instance
+     */
+    @NonNull
+    static Time getCurrent() {
```

Comment:
The point I'm about to make is something we should consider as a stand alone task, not a blocker for this PR. 

In the long run, I think we should discourage people from using the static getter for OS time. If we always pass in the time to constructors (e.g. via the platform context), that means we can always use fake time for any component in a unit test, whenever it becomes convenient. Code that uses the static getter will end up becoming incompatible with tests that need to use fake time.

How exactly we discourage people is something that probably needs discussion.